nss-day-cohort-50/rare-client-rambunctious-rattlers
# Ts comments

## changes made
1. added comment button which pushes to comment form page
2. comments are saved in database after hitting submit button

## how to test
1. open react and go to Post List and click add comment button for any posts
2. to check that they are being added to db you can check the tables in sql or use postman to get comments from /comments
